### PR TITLE
Add instapaper extension

### PR DESCRIPTION
**Pull request by akasuv**
## Description

This extension uses the Instapaper [Simple Developer API](https://www.instapaper.com/api/simple) to save URLs from your clipboard to Instapaper.

## Screencast

https://github.com/user-attachments/assets/6fbb6836-1807-49f8-931e-f43130510a70

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/instapaper/src/save-clipboard.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Clipboard, Toast } from "@raycast/api";
+import { showToast } from "@raycast/api";
+import fetch from "node-fetch";
+import { getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  email: string;
+  password: string;
+}
+
```
Comment (pernielsentikaer) [inline extensions/instapaper/src/save-clipboard.ts:10]:
```suggestion
```

This is already a part of `raycast-env.d.ts` and can be removed

File: extensions/instapaper/package.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  "icon": "instapaper.png",
+  "author": "suv",
+  "categories": [
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "save-clipboard",
+      "title": "Save Clipboard",
```
Comment (pernielsentikaer) [inline extensions/instapaper/package.json:15]:
```suggestion
      "title": "Save Clipboard URL",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by akasuv** [comment]
hi @pernielsentikaer I've updated the code following your suggestion, lmk if anything else needs to be done.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**